src/controllers/plugins/PluginController.cpp

```diff
@@ -306,6 +313,46 @@ const std::map<QString, std::unique_ptr<Plugin>> &PluginController::plugins()
 {
     return this->plugins_;
 }
+std::pair<bool, QStringList> PluginController::updateCustomCompletions(
+    const QString &query, const QString &fullTextContent, int cursorPosition,
+    bool isFirstWord)
+{
+    QStringList results;
+    for (const auto &[name, pl] : getApp()->plugins->plugins())
+    {
+        if (!pl->error().isNull())
+        {
+            continue;
+        }
```

Comment:
This feels like an awkward API - having to check if the plugin has an error in it. Should maybe be handled by the plugin controller instead.
Not something that needs to be fixed in this PR, but maybe some comment should be created for it at least.